minibear/src/main/java/frc/robot/subsystems/WristSubsystem.java

```diff
@@ -0,0 +1,241 @@
+     * TODO - Need this owen
+     */
+    public double getCurrentWristAngle() {
+      return wristEncoder.getPosition() / TICKS_PER_DEGREE;
+    }
+
+    /**
+     * TODO - Need this owen
+     */
+    public WristPosition getTargetWristPosition() {
+      return this.targetWristPosition;
+    }
+
+    /**
+     * TODO - Need this owen
+     */
+    public boolean isPositionMovementComplete() {
+      boolean movementComplete = false;
+      WristPosition targetPosition = this.targetWristPosition;
+      if(targetPosition == WristPosition.None) {
+        movementComplete = true;
+      }
+      else {
+        double targetAngle = this.getWristAngleForPosition(targetPosition);
+        double currentAngle = this.getCurrentWristAngle();
+        double deltaAsPos = Math.abs(targetAngle-currentAngle);
+        movementComplete = (deltaAsPos <= WristSubsystem.ToleranceDegrees);
+      }
+      return movementComplete;
+    }
+
+    /**
+     * TODO - Need this owen
+     * @param position
+     */
+    public void setTargetWristPosition(WristPosition position) {
+      this.setWristAngle(this.getWristAngleForPosition(position));
+    }
+    
+    /**
+     * TODO - Need this owen
+     */
+    @Override
+    public void periodic() {
+      // confirm that the smart motion is setup - no-op after it is setup first time
+      this.initializeMotorsSmartMotion();
+      this.refreshWristPosition();
+      //wristMotor.set(this.requestedWristMotorSpeed * this.wristMotorSpeedReductionFactor);
+
+    }
+
+    /**
+     * TODO - Need this owen
+     */
+    @Override
+    public void simulationPeriodic() {
+
+    }
+
+    /* *********************************************************************
+    PRIVATE METHODS
+    ************************************************************************/
+
+    /**
+     * TODO - Need this owen
+     */
+    private double getWristAngleForPosition(WristPosition position) {
+      double targetWristAngle = Double.NaN;
+      if(position == WristPosition.PickUp) {
+        targetWristAngle = Constants.WRIST_ANGLE_PICKUP;
+      }
+      else if(position == WristPosition.PositionOne) {
+        targetWristAngle = Constants.WRIST_ANGLE_1;
+      }
+      else if(position == WristPosition.PositionTwo) {
+        targetWristAngle = Constants.WRIST_ANGLE_2;
+      }
+      return targetWristAngle;
+    }
+
+    /*
+     * TODO - Need this owen
+     */
+    private void setWristAngle(double wristAngle) { 
+      double targetPositionTicks = wristAngle * TICKS_PER_DEGREE;
+      double curPosition = wristEncoder.getPosition();
+      
+      if (Math.abs(curPosition - targetPositionTicks) > TOLERANCE) {
+          // If the current position is outside the tolerance range, use the PID controller to move the wrist to the target position
+          wristPidController.setReference(targetPositionTicks, ControlType.kPosition);
+      } else {
+          // If within tolerance, stop the wrist motor
+          wristMotor.set(0);
```

Comment:
Owen - I'm wondering if it will droop when we set this?  Should test if break mode is good enough ... or if the motor needs to constantly be attempting to find the target location.